packages/story-editor/src/app/helpCenter/useHelpCenter/effects.js

```diff
@@ -94,10 +94,7 @@ export function deriveUnreadTipsCount(previous, next) {
 }
 
 export function deriveAutoOpen(previous, next) {
```

Comment:
Ignore changes here, this is reverting a PR that causes multiple test failures, see https://github.com/GoogleForCreators/web-stories-wp/pull/10589#issuecomment-1042875858